.travis.yml

```diff
@@ -44,7 +43,7 @@ script:
 after_success:
   - if [ "$COMMAND" == "unit-test" ]; then cat ./coverage/lcov.info | codecov; fi
   - |
-        if [ "$COMMAND" == "end-to-end-test" ] || [ "$COMMAND" == "cockpit-test" ]; then
```

Comment:
This affects code coverage report, right? I. e. this seems unrelated to how welder gets installed into the container. What's the rationale for this? (NB: This may well be correct, I'm not very familiar with the docker-based tests here)

Reply:
The RPM content is not instrumented for code coverage so we can't gather coverage data once this change is applied.

Reply:
Makes sense, thanks.